README.md

```diff
@@ -0,0 +1,30 @@
+> NOTE: We have recently made significant changes to our repository structure. In order to 
+streamline our development process and foster better contributions, we have merged three separate 
+repositories Cumulus, Substrate and Polkadot into this repository. Read more about the changes [
+here](https://polkadot-public.notion.site/Polkadot-SDK-FAQ-fbc4cecc2c46443fb37b9eeec2f0d85f).
+
+# Polkadot SDK
+
+![](https://cms.polkadot.network/content/images/2021/06/1-xPcVR_fkITd0ssKBvJ3GMw.png)
+
+The Polkadot SDK repository provides all the resources needed to start building on the Polkadot 
+network, a multi-chain blockchain platform that enables different blockchains to interoperate and 
+share information in a secure and scalable way. The Polkadot SDK comprises three main pieces of 
+software.
+
+### [Polkadot](./polkadot/)
+
+Implementation of a https://polkadot.network node in Rust based on the Substrate framework. This 
+directory contains runtimes for the Polkadot, Kusama, and Westend networks. 
+
+### [Substrate](./substrate/)
+
+Substrate is the primary blockchain SDK used by developers to create the parachains that make up 
+the Polkadot network.
+
+### [Cumulus](./cumulus/)
+Cumulus is a set of tools for writing Substrate-based Polkadot parachains.
+
```

Comment:
Not sure if we need to mention licensing, but many projects do.  
Each crate should have its own licensing information attached in the `Cargo.toml` and everythin else is `GPL-3.0-only` i assume?

Reply:
We should copy the licensing blurb from the substrate readme I would think.

Reply:
> We should copy the licensing blurb from the substrate readme I would think.

I've added badges to each project to reflect their respective licenses. I don't think it's really necessary to explain Substrate's split license in depth in the root readme. Am I missing something?